pipelines/rj_ssm__celular_seguro/flow.py

```
@@ -0,0 +1,46 @@
+        "client_id": client_id,
+        "client_secret": client_secret
+    }
+    headers = {
+        headers
+    }
+    cert = (
+        cert_crt,
+        cert_key
+    )
+
+    response = requests.post(url,
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)